skopt/space/space.py

```diff
@@ -244,13 +244,13 @@ class Real(Dimension):
     name : str or None
         Name associated with the dimension, e.g., "learning rate".
 
-    dtype : str or dtype, default=np.float
+    dtype : str or dtype, default=np.float64
```

Comment:
Why not just `float`, since that's what the user will become accustomed to seeing?